src/operator/tensor/broadcast_reduce_op_index.cc

```diff
@@ -131,7 +130,7 @@ Examples::
   pick(x, y=[0,1], 0) = [ 1.,  4.]
 
   // picks elements with specified indices along axis 1
-  pick(x, y=[0,1,0], 1) = [ 1.,  4.,  5.]
+  pick(x, y=[0,-1,0], 1) = [ 1.,  4.,  5.]
```

Comment:
Please consider adding extra example for the newly supported case instead of modifying the existing cases.
For this case maybe you can do
```
pick(x, y=[0,1,0,-1], 1) = [ 1.,  4.,  5.,  4.]
```